driver/usb_hid_keys.h

```diff
@@ -0,0 +1,29 @@
+// USB HID keycodes
+// https://source.android.com/devices/input/keyboard-devices.html
```

Comment:
I think it's better to reference e.g. http://www.freebsddiary.org/APC/usb_hid_usages.php

Reply:
OK, I will replace link - I have used first founded one